server/@types/express/index.d.ts

```diff
@@ -19,6 +21,7 @@ export declare global {
     interface Request {
       verified?: boolean
       id: string
+      flash(type: 'errors', message: FlashErrorMessage): number
```

Comment:
When setting, for example, `req.flash('errors', errors)` this stops us needing to add an assertion (e.g. `errors as []`).